### PR TITLE
chore: upgrade php versions and dependencies

### PR DESCRIPTION
**Pull request by houssemzi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 Versions
 ========
 
-2020-11-30
+2020-11-3
```
Comment (mremi) [inline CHANGELOG.md:4]: please keep existing date (matching the next release)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 -----------
-todo ...
+* Upgrade PHP (minor) version to 7.4.12
```
Comment (mremi) [inline CHANGELOG.md:6]:
```suggestion
* Upgrade PHP versions to 7.2.34, 7.3.24 and 7.4.12
```
so only one line :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+* Upgrade PHP (minor) version to 7.3.24
+* Upgrade PHP (minor) version to 7.2.34
+* Upgrade Composer version to 2.0.4
 
-2020-10-31
+2020-10-3
```
Comment (mremi) [inline CHANGELOG.md:11]: please revert this line

File: config.yml [hunk 4]
```diff
@@ -270,7 +270,7 @@ php_test_config: &php_test_config
 php_build_args: &php_build_args
   APCU_VERSION: 5.1.18
   CI_HELPER_VERSION: *CI_HELPER_VERSION
-  COMPOSER_VERSION: 1.10.10
+  COMPOSER_VERSION: 2.0.4
```
Comment (mremi) [inline config.yml:273]: please update other tools as well

**Comment by fpavageau** [comment]
The commit message is not correct:
> preparing 2020/12 release

**Pull request merged**